pkg/v1/providers/infrastructure-vsphere/v0.7.10/ytt/overlay.yaml

```diff
@@ -209,6 +209,36 @@ spec:
       dns:
         imageRepository: #@ kubeadm_image_repo(bomDataForK8sVersion.kubeadmConfigSpec.dns.imageRepository)
         imageTag: #@ bomDataForK8sVersion.kubeadmConfigSpec.dns.imageTag
+      #@ if data.values.TKG_IP_FAMILY == "ipv6,ipv4":
```

Comment:
I'd expect this to produce some clustergen diffs. Are we missing use cases generated to exercise this?

Reply:
Ah, makes sense. We will look at getting some reasonable clustergen diffs.

Reply:
clustergen has apparently been broken for a while now, there are problems with some of the existing constraints. We'll be submitting a separate PR to get them working correctly again.

Reply:
https://github.com/vmware-tanzu/tanzu-framework/pull/1073